.env

```diff
@@ -0,0 +1,30 @@
+STAGE_KEYPAIR_NAME="rhel_stage_id_rsa"
+STAGE_AMI="ami-d2414e38"
+STAGE_INSTANCE_TYPE="t2.micro"
+STAGE_REGION="eu-west-1"
+STAGE_SUBNET_ID="subnet-2c67fe64"
+STAGE_SECURITY_GROUPS="sg-7b78fe07"
+STAGE_SSH_PRIVATE_KEY_PATH="~/.ssh/rhel_stage_id_rsa.pem"
+
+PROD_KEYPAIR_NAME="rhel_aws_id_rsa"
+PROD_AMI="ami-d2414e38"
+PROD_INSTANCE_TYPE="t2.micro"
+PROD_ELASTIC_IP="52.209.230.146"
+PROD_REGION="eu-west-1"
+PROD_SUBNET_ID="subnet-547fcd1c"
+PROD_SECURITY_GROUPS="sg-078605556141c7c6c"
+PROD_SSH_PRIVATE_KEY_PATH="~/.ssh/rhel_aws_id_rsa.pem"
+
+BASE_FOLDER=/vagrant
+NAMESPACE=devopsloft
+DOCKERHUB_APPLICATION=devopsloft
+DOCKERHUB_DB=mysql
+MYSQL_PORT='3306'
+APPLICATION_GUEST_PORT='80'
+APPLICATION_HOST_PORT='5000'
+MYSQL_ROOT_PASSWORD='12345'
+MYSQL_HOST=db
+MYSQL_USER=root
+MYSQL_PASSWORD='12345'
```

Comment:
MYSQL_ROOT_PASSWORD and MYSQL_PASSWORD seems like a duplication

Reply:
[Source](https://dev.mysql.com/doc/refman/5.7/en/docker-mysql-more-topics.html#docker_var_mysql-root-password)
**MYSQL_ROOT_PASSWORD**: This variable specifies a password that is set for the MySQL root account.
MYSQL_USER, **MYSQL_PASSWORD**: These variables are used in conjunction to create a user and set that user's password, and the user is granted superuser permissions for the database specified by the MYSQL_DATABASE variable.

different users for different purposes.